flow/renderer/pyglet_renderer.py

```diff
@@ -391,7 +479,18 @@ def _add_vehicle_poly_circle(self, center, radius, color):
             vertex_color += color
         index = [x for x in range(pxpm)]
         group = pyglet.graphics.Group()
-        self.vehicle_batch.add_indexed(pxpm, pyglet.gl.GL_LINE_LOOP,
-                                       group, index,
-                                       ("v2f", vertex_list),
-                                       ("c3B", vertex_color))
+        self.vehicle_batch.add_indexed(
+            pxpm, pyglet.gl.GL_LINE_LOOP, group, index,
+            ("v2f", vertex_list), ("c4B", vertex_color))
+
+    def _truncate_colormap(self,
+                           cmap,
+                           minval=0.25,
+                           maxval=0.75,
+                           n=100):
+        """Truncate a matplotlib colormap."""
```

Comment:
Maybe add what each parameter is? most of them are clear, but `n` is confusing. Alternatively, you can just give `n` a clearer name and then describing the parameters shouldn't be necessary

Reply:
Docs added.